src/connectRoutes.js

```diff
@@ -115,8 +115,8 @@ export default (
   }
 
   const {
-    location: locationKey = 'location',
-    title: titleKey = 'title',
+    selectLocationState = state => state.location,
```

Comment:
Ok, to be clear, were you thinking to overload the `title` and `location` options to allow functions and strings or to keep them around in addition to `selectLocatin/TitleState` ?